philology/src/test/java/com/jcminarro/philology/transformer/SupportToolbarViewTransformerTest.kt

```diff
@@ -70,4 +77,44 @@ class SupportToolbarViewTransformerTest {
         Verify on toolbar that toolbar.setSubtitle(1) was called
         `Verify no further interactions` on toolbar
     }
-}
\ No newline at end of file
+
+    @Test
+    fun `Should reword only the title from style`() {
```

Comment:
What happen if a title is set on the xml and into the style?
Which one should be used?
Could we create a test about it?

Reply:
The same with subtitle

Reply:
Turns out system pick the one defined in the xml attributes ignoring `styles` in that case. Adjusted implementation according to that.